Source/Resources/Content/MSGraph/Groups/Baseline - Corporate Android Devices.json

```diff
@@ -0,0 +1,13 @@
+{

```

Comment:
should follow existing convention where the platform is at the end (e.g. Corporate Devices - Physical)